operator/controllers/keptntask/job_utils.go

```diff
@@ -117,10 +117,8 @@ func (r *KeptnTaskReconciler) updateJob(ctx context.Context, req ctrl.Request, t
 	}
 	if job.Status.Succeeded > 0 {
 		task.Status.Status = common.StateSucceeded
-		err = r.Client.Status().Update(ctx, task)
-		if err != nil {
-			r.Log.Error(err, "could not update job status for: "+task.Name)
-		}
```

Comment:
why we removed the status update?

Reply:
because the Status is updated afterwards by the Reconciler which calls this function: https://github.com/keptn/lifecycle-toolkit/blob/5bd4e4c80d0c59a8a1aa5f22b1b72ced82169178/operator/controllers/keptntask/controller.go#L83